components/sbm-recipes-boot-upgrade/src/main/resources/recipes/boot-autoconfiguration-update.yaml

```diff
@@ -0,0 +1,11 @@
+- name: boot-autoconfiguration-update
+  description: Create org.springframework.boot.autoconfigure.AutoConfiguration.imports file for new spring 2.7
+  condition:
+    type: org.springframework.sbm.common.migration.conditions.TrueCondition
```

Comment:
The condition for the recipe should better be `IsMatchingSpringBootVersion` with pattern `2.7.` so it is only shown for 2.7 Boot applications.